bodhi/client/__init__.py

```diff
@@ -370,11 +370,22 @@ def edit(user, password, url, **kwargs):
             resp = client.query(**query_param)
             title = resp['updates'][0]['title']
         elif re.search(bindings.UPDATE_TITLE_RE, kwargs['update']):
+            query_param = {'like': kwargs['update']}
+            resp = client.query(**query_param)
             title = kwargs['update']
         del(kwargs['update'])
         kwargs['builds'] = title
         kwargs['edited'] = title
 
+        # Convert list of 'Bug' instances in DB to comma separated bug_ids for parsing.
+        former_update = resp['updates'][0]
+        kwargs['bugs'] = ",".join([str(bug['bug_id']) for bug in former_update['bugs']])
```

Comment:
You need to do this in the `for field in kwargs` loop after checking that `kwargs['bugs'] is None` otherwise this will delete bugs that the user gave using the CLI.

Reply:
Keeping check inside the for loop will add an extra if and that will repeat multiple times. Can we do a check separately outside like this?

```if not kwargs['bugs']:
           kwargs['bugs'] = ",".join([str(bug['bug_id']) for bug in former_update['bugs']])```

Reply:
Yes I think that would work :+1: